src/controller/parsing/parsing.py

```
@@ -0,0 +1,85 @@
+
+
+
+# Не знаю, как правильно юзать Depends и главное для чего
+@router.post("") #, response_model=UploadCardRequest) 
+@version(1)
+def create_upload_card(request: UploadCardRequest):
+    try:
+        # Извлекаю номенклатуры из pdf файла по ссылке
+        link_pdf_file = request.documents[0]
+        uuid = request.guid
+        nomenclatures = parsing_upd_file(link=link_pdf_file, uuid=uuid)
```

Ruff (0.8.0)

Local variable `nomenclatures` is assigned to but never used

Remove assignment to unused variable `nomenclatures`

(F841)